### DISCUSSION TITLE
Student Benefits approved

### DISCUSSION THREAD

**Discussion by iSMP14** [community/community#75650] [GitHub Education]
### Hi everyone,

Hi everyone!! 
i have a issue; i apply for the student program and its approved, but i dont have access to the benefits :c 

![Screenshot_3](https://github.com/community/community/assets/132416768/99ba89a0-df50-42cb-8f29-044b6c2d9d72)
![Screenshot_4](https://github.com/community/community/assets/132416768/417be864-165a-44c0-a0fb-de53893e0c91)

**Comment by jamirou** [comment]
The waiting period is approximately one week minimum. If you sent the form and your data on November 12 you should wait a little longer :) I hope that helps friend.
